spec/computer_spec.rb

```diff
@@ -1,75 +1,87 @@
-require 'computer.rb'
-require 'game.rb'
-require 'board.rb'
+require File.join(File.dirname(__FILE__), '..', 'lib', 'computer')
+require File.join(File.dirname(__FILE__), '..', 'lib', 'game')
+require File.join(File.dirname(__FILE__), '..', 'lib', 'board')
```

Comment:
esto no fue lo que vimos del require, el chiste es no reforzar la estructura de tus directorios; esto sigue haciendo lo mismo pero mas largo

https://thoughtbot.com/blog/following-the-path

Reply:
Done